roles/rsyslog_client/tasks/client.yml

```diff
@@ -0,0 +1,158 @@
+---
+- name: Get information on local CA certificate
+  ansible.builtin.stat:
+    path: "{{ rsyslog_local_dir }}/{{ rsyslog_ca_cert_file }}"
+    checksum_algorithm: sha1
```

Comment:
Checksum algorithm is specified, which suggests we want to validate that checksum later on, but I cannot find any checksum validation...